src/components/contact/contact.js

```diff
@@ -0,0 +1,104 @@
+import React from "react"
+import { H2, Body, Sub2 } from "../../style/type-styles"
+import {
+  ContactContainer,
+  ContactForm,
+  ContactSection,
+  ContactDescription,
+  ContactButton,
+  ContactImageSection,
+  SpecificContactInfo,
+  EmailSection,
+  AddressSection,
+} from "./contact-styles"
+import PlantIcon from "../../vectors/plant-icon.svg"
+
+export default function Contact({ data }) {
+  return (
+    <ContactContainer>
+      <ContactSection>
+        <ContactForm>
+          <PlantIcon />
+          <H2>{data.prismicContactPage.data.overview_heading}</H2>
+          <ContactDescription>
+            <Body>{data.prismicContactPage.data.overview_description}</Body>
+          </ContactDescription>
+          <form>
+            <label>
```

Comment:
Need form labels here:
https://www.w3.org/WAI/tutorials/forms/

Also the typography does not match the hi-fi designs.